api-specificatie/openapi.yaml

```diff
@@ -116,35 +183,29 @@ paths:
 
   # JBO: Het ontsluiten van een adres op basis van de unieke adres-id wordt gebruikelijk gedaan met een path-parameter. Dat impliceert wel een unieke identificatie en is dus geen zoeken meer maar raadplegen. Dat klopt met de response definitie waarin geen array, maar een enkel adres wordt geretourneerd
   #JBO: Krijgen alle adressen nu een eigen adres-id of gaan we de nummeraanduiding-id gebruiken als adres-id ?
-  #MSR: adressen kunnen geen eigen ID hebben (afgezien van een nummeraanduiding), omdat adressen worden samengesteld uit meerdere andere objecten. Als adressen een eigen ID zouden krijgen, zouden deze ook gepersisteerd moeten worden en dat gebeurt niet, modat adres geen IMBAG object is. 
+  #MSR: adressen kunnen geen eigen ID hebben (afgezien van een nummeraanduiding), omdat adressen worden samengesteld uit meerdere andere objecten. Als adressen een eigen ID zouden krijgen, zouden deze ook gepersisteerd moeten worden en dat gebeurt niet, omdat adres geen IMBAG object is. 
   /adressen/{nummeraanduidingIdentificatie}:
     get:
       security:
         - apiKeyBAG: []
-      operationId: zoekAdres
+      operationId: raadpleegAdresMetNumId
       tags:
         - Adres
       summary: Raadpleeg adres o.b.v. een sleutel
-      description: "Raadpleeg het adres op basis van de nummeraanduiding identificatie verkregen via de zoeken-operatie
+      description: "Raadpleeg het adres met de nummeraanduiding identificatie verkregen via de zoek-operatie
```

Comment:
De nummeraanduidingIdentificatie krijg je niet (meer hebben we besloten) met de zoekoperatie. Dit is precies wat we bedoelen. Als je te ver gaat in het beschrijven van de werking van je implementatie, klopt het niet meer als je implementatie verandert :-)!

Reply:
Issue #60 aaangemaakt.